dingo/gurobi_based_implementations.py

```diff
@@ -273,6 +274,601 @@ def fast_fva(lb, ub, S, c, opt_percentage=100):
+                                beq_res = np.append(
+                                    beq_res, min(max_objective, min_objective)
+                                )
+                                # Remove the bounds on this dimension
+                                ub[i] = sys.float_info.max
+                                lb[i] = -sys.float_info.max
+                            else:
+                                # store this dimension
+                                indices_iter.append(i)
+
+                                if not redundant_facet_left:
+                                    # Not a redundant inequality
+                                    A_res = np.append(
+                                        A_res,
+                                        np.array(
+                                            [
+                                                A[
+                                                    n + i,
+                                                ]
+                                            ]
+                                        ),
+                                        axis=0,
+                                    )
+                                    b_res.append(b[n + i])
+                                else:
+                                    lb[i] = -sys.float_info.max
+
+                                if not redundant_facet_right:
+                                    # Not a redundant inequality
+                                    A_res = np.append(
+                                        A_res,
+                                        np.array(
+                                            [
+                                                A[
+                                                    i,
+                                                ]
+                                            ]
+                                        ),
+                                        axis=0,
+                                    )
+                                    b_res.append(b[i])
+                                else:
+                                    ub[i] = sys.float_info.max
+                        else:
+                            # Remove the bounds on this dimension
+                            ub[i] = sys.float_info.max
+                            lb[i] = -sys.float_info.max
+
+                b_res = np.asarray(b_res)
+                A_res = np.asarray(A_res, dtype="float")
+                A_res = np.ascontiguousarray(A_res, dtype="float")
+
+                return (
+                    A_res,
+                    b_res,
+                    Aeq_res,
+                    beq_res,
+                )
+
+    # Print error messages
+    except gp.GurobiError as e:
+        print("Error code " + str(e.errno) + ": " + str(e))
+    except AttributeError:
+        print("Gurobi solver failed.")
+
+
+def fast_find_redundant_facets_2(lb, ub, S, c, opt_percentage=100):
```

Comment:
what is the difference from the previous function? is this some optimized second version?

Reply:
That was for debugging. Deleted